pkg/network/encoding/encoding.go

```diff
@@ -69,20 +101,12 @@ func modelConnections(builder *model.ConnectionsBuilder, conns *network.Connecti
 	})
 
 	routeIndex := make(map[string]RouteIdx)
-	httpEncoder := newHTTPEncoder(conns.HTTP)
-	defer httpEncoder.Close()
-	kafkaEncoder := newKafkaEncoder(conns.Kafka)
-	defer kafkaEncoder.Close()
-	http2Encoder := newHTTP2Encoder(conns.HTTP2)
-	defer http2Encoder.Close()
 
-	ipc := make(ipCache, len(conns.Conns)/2)
-	dnsFormatter := newDNSFormatter(conns, ipc)
 	tagsSet := network.NewTagsSet()
```

Comment:
`tagSet` is not also a `ConnectionsModeler` level?

Reply:
Same for `routeIndex`

Reply:
added